lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -177,13 +177,13 @@ def self.create_github_release(release_version, release_description, upload_asse
         )
       end
 
-      private_class_method def self.replace_in(previous_text, new_text, path, allow_empty: false)
+      def self.replace_in(previous_text, new_text, path, allow_empty: false)
         if new_text.to_s.strip.empty? && !allow_empty
           UI.user_error!("Missing `new_text` in call to `replace_in`, looking for replacement for #{previous_text} 😵.")
         end
         sed_regex = "s|#{previous_text.sub('.', '\\.')}|#{new_text}|"
         backup_extension = '.bck'
-        Actions.sh("sed", '-i', backup_extension, sed_regex, path)
+        Actions.sh("sed", "-i#{backup_extension}", sed_regex, path)
```

Comment:
TIL that `sed` works differently on macosx and linux... I thought about writing code for this instead of doing it using a command, but thought it would be much more inefficient, specially for large files. In the end, I found this: https://stackoverflow.com/a/22084103. I'm not a terminal guru, but I tested it in my machine and in the circleci machines and it worked perfectly fine. Please take a look to make sure this is ok!

Reply:
honestly we might just want to go full ruby rather than `sed`, which will also prevent OS differences

Reply:
Yeah... I was worried about performance in some of those `pbxproj` iOS files... But I think it should probably be fine... I might just run some tests tomorrow and make sure but would probably be cleaner than this.

Reply:
I moved this extra implementation to https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/7 since this PR was already approved and I didn't want to change implementation further. I checked performance and didn't run into issues even when trying to substitute somewhat large files.